CMakeLists.txt

```diff
@@ -332,7 +332,7 @@ else()
 endif()
 
 set( _DEST "${_DESTDIR}" )
-set( _PREFIX "${CMAKE_INSTALL_PREFIX}" )
+set( INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}" )
```

Comment:
Needed for [audacity_config.h.in](https://github.com/audacity/audacity/blob/c6ef82360a66d6fde071b546a44b566df4fadc3f/src/audacity_config.h.in#L36).

Reply:
Understood:  the .in file doesn't change but was assuming assignment of a Cmake variable had happened, when it had not.